app/src/main/java/com/epam/LessonsActivity.java

```diff
@@ -5,29 +5,81 @@
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
 
 import com.epam.cleancodetest.R;
 import com.epam.themes.androidcomponents.AndroidComponentsActivity;
+import com.epam.themes.cleancode.CleanCodeActivity;
 import com.epam.themes.uicomponents.UIComponentsActivity;
 
 public class LessonsActivity extends AppCompatActivity {
 
+    private ViewGroup firstLesson;
+    private ViewGroup secondLesson;
+    private ViewGroup thirdLesson;
+
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_lessons);
 
-        findViewById(R.id.lesson_components_root_view).setOnClickListener(new View.OnClickListener() {
+        firstLesson = findViewById(R.id.fist_lesson);
+        secondLesson = findViewById(R.id.second_lesson);
+        thirdLesson = findViewById(R.id.third_lesson);
+
+        setLessonParameters(firstLesson, R.drawable.ic_mentor_face, R.string.mentor_name_yahor,
```

Comment:
Instead of passing 4 params to method create a class to hold this params.

Reply:
done